contributing.md

```diff
@@ -175,3 +175,28 @@ You may then view `output.log` in VSCode. A typical log line is then:
 ```log
 [14:37:36.517] DEBUG <engine.go:114>: 0x111A00  <  {"engine":"0x111A00","message":"Constructed Engine"}
 ```
+
+### Start RPC servers with Docker
+
+To spin up a docker image with 3 rpc servers and channels pre-populated, run the following:
+
+1. `make docker/build`
+1. `make docker/start`
+
+Three rpc go-nitro servers will be available on ports 4005, 4006, and 4007 for Alice, Irene, and Bob. A ledger channel is created between Alice and Irene, and another ledger channel is created between Irene and Bob. A virtual channel is created between Alice and Bob.
+
```

Comment:
It's not obvious at this point why you might do this. Maybe adding some sample CLI calls to each client?

... I don't know how to do this!